recipe/meta.yaml

```diff
@@ -1,58 +1,70 @@
 {% set name = "hydra-core" %}
-{% set version = "1.3.1" %}
+{% set version = "1.3.2" %}
 
 package:
   name: {{ name|lower }}
   version: {{ version }}
 
 source:
-  url: https://github.com/facebookresearch/hydra/archive/v{{ version }}.tar.gz
-  sha256: 9ba1efe09893853e883498fcbeec17f2a77e1ba551587d6d0604077e02de1602
+  - url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+    sha256: 8a878ed67216997c3e9d88a8e72e7b4767e81af37afb4ea3334b269a4390a824
+    patches:
+    # do not forget to update the version literal in the antlr-*-complete.jar string inside the
+    # patch to correctly unvendor antlr and use the antlr package from conda
+      - 0000-unvendor_antlr.patch
+    folder: pypi-upstream
```

Comment:
could you please use the root folder for this? (remove the `folder` tag, and update the `license_file` path)

Reply:
this was due to an old suggestion from @JeanChristopheMorinPerso, that whenever we have both a PyPi and a GitHub tarball, because we give priority to the PyPi for the actual package build and only fetch the GitHub tarball to have some test assets that are not present in the PyPi archive, we put these in separate directories to prevent the directory where the GitHub archive gets extracted to overwrite a directory in the PyPi archive, which would end up in the same path.
Correct me if you stand by this @JeanChristopheMorinPerso. I may have misunderstood.

I guess that if we check the content of the PyPi archive and give the `test-assets` directory a proper name that doesn't conflict, we shouldn't have to worry about this, but at the same time this way we will always be sure of not running into conflicting names.

Reply:
It would be enough to have the GH archive for the tests + data files in a folder such as gh_src and the pypi archive in the root. You would avoid using `%SRC_DIR%` for win and `$SRC_DIR` for the rest.